process/sync/baseSync.go

```diff
@@ -442,6 +454,16 @@ func (boot *baseBootstrap) ShouldSync() bool {
 
 	boot.roundIndex = boot.rounder.Index()
 
+	if boot.statusHandler != nil {
```

Comment:
delete this if, if you have initialized the statusHandler in the constructor to nil status handler.

Reply:
done